src/index.html

```diff
@@ -16,17 +16,36 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <label class="search-bar">
+      <form
+        method="get"
+        data-qa="big"
+        class="search-bar__big"
+      >
+        <input
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          class="search-bar__input search-bar__input--big"
+        />
+      </form>
+    </label>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <label class="search-bar">
+      <form
+        method="get"
+        data-qa="big"
```

Comment:
```suggestion
        data-qa="small"
```
it should fix such problem
![Screenshot 2024-04-10 at 23 34 25](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/4f4efd2e-0271-4fc5-9158-90b57f226276)